File-Formats/DocIO/Convert-Word-to-PDF-in-Azure-function-v4.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Convert Word document to PDF in Azure Functions v4 | Syncfusion
+description: Convert Word document to PDF in Azure Functions v4 using .NET Core Word (DocIO) library without Microsoft Word or interop dependencies.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word document to PDF in Azure Functions v4
+
+Syncfusion DocIO is a [.NET Core Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read,  edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can perform **convert a Word document to PDF in Azure Functions v4**
```

Comment:
Using this library, you can **convert a Word document to PDF in Azure Functions v4**

Reply:
Updated